app/team/join-the-team/page.tsx

```diff
@@ -13,274 +13,264 @@ import FAQSection from "../../components/FAQSection";
 import { List, Space } from "antd";
 
 const memberTasks = [
-    "Part Design",
-    "Manufacturing",
-    "Testing",
-    "Marketing/Fundraising",
-    "Data Analysis",
-    "Simulation Modeling",
+  "Part Design",
+  "Manufacturing",
+  "Testing",
+  "Marketing/Fundraising",
+  "Data Analysis",
+  "Simulation Modeling",
 ];
 
 const joinTheTeam = [
-    "1. Join our discord server by filling out this form, or come by the shop (ECSW 1.455)",
-    "2. Join the channel(s) for the subteam(s) you are interested in",
-    '3. Introduce yourself and ask any questions such as "How can I get involved? Are there any projects that need an extra hand? Here are my skills…"',
-    "4. Find meeting dates for subteam meetings under the #about channel",
-    "5. Attend the meeting and get started on projects!",
+  "1. Join our discord server by filling out this form, or come by the shop (ECSW 1.455)",
+  "2. Join the channel(s) for the subteam(s) you are interested in",
+  '3. Introduce yourself and ask any questions such as "How can I get involved? Are there any projects that need an extra hand? Here are my skills…"',
+  "4. Find meeting dates for subteam meetings under the #about channel",
+  "5. Attend the meeting and get started on projects!",
 ];
 
 const becomeALead = [
-    "Being well-versed in vehicle dynamics",
-    "Understanding system limits",
-    "Having an effective engineering process",
-    "Being an effective teammate",
-    "Working closely with other subsystems and leads",
-    "Having proficient communication skills with your team",
-    "Being an active and hard-working member of DFR",
-    "Taking initative on projects",
-    "Contributing unique interpretations withing FSAE limits",
-    "Commiting more time to the team throughout the year",
-    "Attending team volunteeting opportunities",
-    "Taking on independent projects",
-    "Learning new or needed skills for bettering the team",
+  "Being well-versed in vehicle dynamics",
+  "Understanding system limits",
+  "Having an effective engineering process",
+  "Being an effective teammate",
+  "Working closely with other subsystems and leads",
+  "Having proficient communication skills with your team",
+  "Being an active and hard-working member of DFR",
+  "Taking initative on projects",
+  "Contributing unique interpretations withing FSAE limits",
+  "Commiting more time to the team throughout the year",
+  "Attending team volunteeting opportunities",
+  "Taking on independent projects",
+  "Learning new or needed skills for bettering the team",
 ];
 
 const text = `something`;
 
 const FAQ = [
```

Comment:
Maybe later on we can create just a JoiningFAQ component so we don't need to move this around.